RELEASE_NOTES.md

```diff
@@ -1,3 +1,12 @@
+### Version 2.2.6 (2017)
+
+* Add Tomcat 8.5.x alongside 8.0.x
+* Updated tomcat versions to latest release version
+* Requires tomcat configuration in build.gradle:
```

Comment:
If it is something specific to Tomcat 8.5.x why not make that the default in the Tomcat 8.5.x implementation?